etcdutils.go

```diff
@@ -152,28 +148,29 @@ func RemoveCerts(etcdStaticResourceDir string) {
 	}
 }
 
-func RestoreSnapshot() {
-	// NOT needed, direct etcdctl call
-}
+func PatchManifest(manifestFilePath, old, new string) {
 
-func PatchManifest() {
-}
+	read, err := ioutil.ReadFile(manifestFilePath)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%s:read error %s", manifestFilePath, err)
+	}
+	fmt.Println(manifestFilePath)
 
-func EtcdMemberAdd() {
-	// Direct call to etcdctl
+	newContents := strings.Replace(string(read), old, new, -1)
+
+	err = ioutil.WriteFile(manifestFilePath, []byte(newContents), 0)
```

Comment:
style nit: this is the case when you write `if err:= ...; err !=nil {}` ... you don't need the `err` below